doc/source/releases/index.rst

```diff
@@ -37,3 +37,15 @@ Releases
    13.02_announce
    13.01_announce
    older_releases
+
+User Announcements
+
+.. toctree::
+   :maxdepth: 1
+
+   20.09_announce_user
```

Comment:
I don't like the duplication of announcements, we could replace the XX.XX_announce with XX.XX_announce_user above for the releases that have them.

Reply:
These are developer release notes right? Don't we intentionally not lead with the user release notes? If we wanted to lead with user release notes - we should just not produce developer release notes?

If you don't link these links here - what about another page with the user release notes?

Reply:
> These are developer release notes right?

By looking at the docs menu, that's the general (and only at the moment) release notes page, if it was for developer release notes it should go under "Developer Documentation", I think.

> Don't we intentionally not lead with the user release notes?

Maybe, I don't recall the discussion though, it could be a good time to rethink given the increasing effort on the user RNs.

> If we wanted to lead with user release notes - we should just not produce developer release notes?

I think this page should list the user release notes (`XX_announce` for 18.09 and earlier), and all user release notes already link to the detailed/dev-oriented release notes at the bottom.

Ping @hexylena

Reply:
I didn't push them as a replacement since they're different audiences, and I think devs like reading the dev notes for a rollup of the past N months work.

That said I can definitely see the duplication in the menu being suboptimal.

I guess the question is: are users (or in general people who would benefit from seeing these) finding their way to docs.galaxyproject.org? I'm not sure at all.  They are mentioned in the dev docs, but, argh.

Reply:
> are users (or in general people who would benefit from seeing these) finding their way to docs.galaxyproject.org? I'm not sure at all. They are mentioned in the dev docs, but, argh.

We are now directly linking to the user release notes from the Galaxy Help menu: https://github.com/galaxyproject/galaxy/pull/10960

Reply:
I don't like it in the abstract but I don't feel strongly enough to fight on this so https://github.com/galaxyproject/galaxy/pull/11103/commits/ba7286ef45c6bc9420d55f163ce126c45826119a. Also what I do feel strongly about is the last release's user release notes were shockingly good. I hadn't seen them before yesterday and I was blown away by the effort that must have been put into them - so I have no issue with making them more prominent.

Reply:
```
/Users/john/workspace/galaxy/doc/source/releases/19.05_announce.rst: WARNING: document isn't included in any toctree
/Users/john/workspace/galaxy/doc/source/releases/19.09_announce.rst: WARNING: document isn't included in any toctree
/Users/john/workspace/galaxy/doc/source/releases/20.01_announce.rst: WARNING: document isn't included in any toctree
/Users/john/workspace/galaxy/doc/source/releases/20.05_announce.rst: WARNING: document isn't included in any toctree
/Users/john/workspace/galaxy/doc/source/releases/20.09_announce.rst: WARNING: document isn't included in any toctree
```

Well that didn't even fix that problem 😓 . This PR got away from itself.

Reply:
So should we mark as orphan the XX.XX_announce.rst files for 19.01 and following ?

Reply:
Same comment as above - I think we should merge this while it is green and re-tackle release issue stuff in a clean PR focused on that.